lib/rules/color-no-invalid-hex/__tests__/index.js

```diff
@@ -61,6 +61,10 @@ testRule({
 				'font-style: normal;\n' +
 				'}',
 		},
+		{
+			code: 'a { color: #colors[somecolor]; }',
+			description: 'Less map usage',
+		},
```

Comment:
Let's move this into its own `testRule` case (towards the bottom of the file), which sets to syntax to `postcss-less` as `testRule` without syntax should only include vanilla CSS constructs.

Reply:
👍  Sure, as soon as I have at least some time.